constants/paths.js

```diff
@@ -0,0 +1,14 @@
+const path = require("path");
+
+exports.S3_DIR = {
+  VIDEOS: {
+    MAIN: path.join(__dirname, "../temp/videos/main/"),
+    SUB_ONE: path.join(__dirname, "../temp/videos/sub-one"),
+    SUB_TWO: path.join(__dirname, "../temp/videos/sub-two"),
+  },
+  AUDIOS: {
+    MAIN: path.join(__dirname, "../temp/audios/main/"),
+    SUB_ONE: path.join(__dirname, "../temp/audios/sub-one"),
+    SUB_TWO: path.join(__dirname, "../temp/audios/sub-two"),
+  },
+};
```

Comment:
경로를 상수 처리해주셔서 사용시 용이할 것 같습니다 👍 
다만 현재 클라이언트 측에서 사용되는 zustand 스토어 값과 서버측 용어가 달라 이부분 추후에 한쪽의 변경이 필요할 것 같습니다!

또한 S3의 디렉토리가 아닌, 서버에서 처리를 위한 temp 디렉토리므로 해당 부분 S3_DIR 보다는 `TEMP_DIR` 혹은 다른 명칭으로 변경하는 것이 좋을 것 같습니다.

추가로, 하나의 `S3_DIR`에서 모든걸 처리하는 것처럼 보이는데, 현재 프로젝트에서는 오디오와 비디오를 동시에 참조해야할 부분이 많지 않아서 별도로 분리하는 것이 사용하기에 용이할 것 같다는 의견입니다 :)

```suggestion
exports.TEMP_DIR_VIDEOS = {
  MAIN: path.join(__dirname, "../temp/videos/main/"),
  SUB_ONE: path.join(__dirname, "../temp/videos/sub-one"),
  SUB_TWO: path.join(__dirname, "../temp/videos/sub-two"),
};
exports.TEMP_DIR_AUDIOS = {
  MAIN: path.join(__dirname, "../temp/audios/main/"),
  SUB_ONE: path.join(__dirname, "../temp/audios/sub-one"),
  SUB_TWO: path.join(__dirname, "../temp/audios/sub-two"),
};
```

Reply:
의견 감사드립니다! 이름 변경과 위와 같이 분리하도록 하겠습니다!